ClickToCall.Web.Tests/packages.config

```diff
@@ -5,11 +5,11 @@
   <package id="Microsoft.AspNet.WebPages" version="3.2.3" targetFramework="net452" />
   <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net452" />
   <package id="Moq" version="4.2.1510.2205" targetFramework="net452" />
-  <package id="Newtonsoft.Json" version="6.0.4" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="8.0.1" targetFramework="net452" />
```

Comment:
Perhaps this could be updated to Newtonsoft.Json 9.0.1 to make it consistent throughout the solution?